### ISSUE TITLE
Update header component

### ISSUE THREAD

**Issue by cmajel** [cloud-gov/cg-site#1541]
In our UX audit, we noted a few issues with the header component, including:

* The active nav item should be noted
* The search input should be visible with a label readable by screen readers.
* Search is not shown on mobile.
* The top nav doesn't collapse on mobile.
* The top nav and side nav on mobile is confusing, as it creates a dueling nav situation. 
* Some top nav items, like login and platform status, are external links and do not navigate you within the site

Note that some of these issues are closely tied to the site IA, and may be influenced by how we organize the site's content.

## Done when
* [x] Updated header design is shared
* [x] Discuss with team, and determine what can be acted on now 
* [x]  Implement updates on the site

## Notes
* UX audit [header section](https://docs.google.com/document/d/1ItGl3CKfGf9BHPDA78UOLFNnrpOYl3TJC3y6c-K7JT8/edit#heading=h.nvjs5p44hs9j) and [search section](https://docs.google.com/document/d/1ItGl3CKfGf9BHPDA78UOLFNnrpOYl3TJC3y6c-K7JT8/edit#heading=h.sss7hp24h83u)

**Comment by cmajel** [comment]
@lauraGgit here's what I have so far for the header section. A few notes:

## Content
Supernav
* Platform status
* Login

Main nav
* Sign up
* Pricing
* Documentation
* Updates
* Contact



## General spacing
* I'm using the USWDS 8px spacing units, so most spacing is a multiple of 8 (8px, 16px, 48px etc) 

## Desktop
* Main header items text styles:  18Franklin Medium, `base-ink` color, 14px / `2xs`. 
  * Items are 48px apart. 
* Supernav text styles: 18Franklin Regular, `base-ink` color, 13px / `3xs` 
* Borders are `1px solid $base-light / #C6CACE`
* On hover, text color switches to `primary-vivid`
* Active item bar is 5px height and `primary-vivid`
![cloudgov--header-1](https://user-images.githubusercontent.com/2374206/74245923-c5392380-4cb1-11ea-9ca5-a85875604547.jpg)

![cloudgov--header-1--grid](https://user-images.githubusercontent.com/2374206/74245487-13015c00-4cb1-11ea-97f5-eeb3901dfe78.png)

## Mobile
* Stealing basic pattern from [USWDS basic header](https://federalist-proxy.app.cloud.gov/preview/uswds/uswds/release-2.4.0/components/preview/header--basic.html/)
* All nav items text styles: :  18Franklin Medium, `base-ink` color, 14px / `2xs`

![cloudgov--header-1--mobile](https://user-images.githubusercontent.com/2374206/74183289-ae8fbf80-4c12-11ea-8516-615b2602aadb.png)

![cloudgov--header-1--mobile--grid](https://user-images.githubusercontent.com/2374206/74245501-18f73d00-4cb1-11ea-8b21-02092381b8d6.png)


I am still not totally sure how the top nav and side nav interact in this world, especially since the top level sidnav items are categories rather than pages (for example, there isn't an `Overview` page). I suppose we could do a [mega menu with category headers](https://federalist-proxy.app.cloud.gov/preview/uswds/uswds/release-2.4.0/components/preview/header--extended-mega.html), but this would be a lot for the top nav. 

But maybe we can pair on this.

**Comment by cmajel** [comment]
@lauraGgit updated above comment with the grid view and a content section where you can select the text.

**Comment by cmajel** [comment]
Updated grid graphic for a 1024px grid 
![Screen Shot 2020-02-11 at 4 43 19 PM](https://user-images.githubusercontent.com/2374206/74282219-c25c2400-4ced-11ea-8380-ede4bdb01849.png)

**Comment by cmajel** [comment]
@lauraGgit doing a final styling review on this and passing to @karareinsel for her input.

Current on [preview site](https://cg-4bc824cf-0e89-4d31-a1be-d64b4c9fec53.app.cloud.gov/preview/18f/cg-site-federalist/colors-and-content/):
![Screen Shot 2020-02-18 at 12 59 23 PM](https://user-images.githubusercontent.com/2374206/74763903-a527d800-524e-11ea-9030-a3b1f0aa98ac.png)
### Styling 
* The default link color should be `base-ink` / `#171716` / `gray-warm-90` from [the color palette](https://github.com/18F/cg-site/issues/1534). Currently it's `#2e2e2a` 
* The border on the search bar should be set to `base-light` / `#C6CACE` / `gray-cool-20`  from [the color palette](https://github.com/18F/cg-site/issues/1534). Currently it's `#565C65`
* We talked about this color shift for text links but it also applies for the header. On hover the text for nav items should be set to `primary-dark` /  `#0050D8` / `blue-warm-60v` from [the color palette](https://github.com/18F/cg-site/issues/1534).. The indicator bar can remain `primary-vivid`
#### Mobile

Current:
![Screen Shot 2020-02-18 at 1 18 07 PM](https://user-images.githubusercontent.com/2374206/74765263-3d26c100-5251-11ea-9480-30e4043c03bc.png)
![Screen Shot 2020-02-18 at 1 17 57 PM](https://user-images.githubusercontent.com/2374206/74765261-3c8e2a80-5251-11ea-931a-6065d16658ee.png)
* Search input is not fully visible on mobile
* Can we add 16px / `2` USWDS spacing unit token to the top/bottom padding of the logo? It is riding pretty closely to the border and government banner. 
* I know this is on your radar, but the mobile dropdowns are white on white and not visible.


### Content
* We don't need a `Home` link in the top nav, as the logo is mapped to home and (I believe) this would be repetitive for users of screen readers.
* Under `Why cloud.gov`, our team currently maps to the home page, which doesn't seem right. Do we have a destination for this page? If not, maybe it should be removed for now?

@karareinsel can you review for any additional style and content notes?

**Comment by karareinsel** [comment]
Content
- Totally fine with removing the Home link but we should double check that the cloud.gov logo takes people to the home page. If you click on the logo on the site in production, it just keeps you on whatever page you're on.
- Why cloud.gov should map to https://cloud.gov/docs/overview/cloudgov-team/
- When I click on We're FedRAMPed Authorized, it takes me to that page but the left nav isn't open. When I click on other pages under Why Cloud.gov? the left nav is open when I land on the selected page.
- Officially it's cloud.gov (lowercase c) - the hero image on Get Started uses a capital C

**Comment by karareinsel** [comment]
Mobile
- The hero imagery on Get Started needs some adjusting›
<img width="502" alt="Screen Shot 2020-02-18 at 4 36 14 PM" src="https://user-images.githubusercontent.com/34074929/74780070-de6f4080-526c-11ea-81e5-1c741b9bf234.png">
- The table on the pricing page is not working well in a mobile environment - the lines of data are stacking on top of one another so the information no longer makes sense Imagery and text weirdness on pricing page 
<img width="501" alt="Screen Shot 2020-02-18 at 4 37 44 PM" src="https://user-images.githubusercontent.com/34074929/74780151-02328680-526d-11ea-9afe-6362caac2f7d.png">
<img width="504" alt="Screen Shot 2020-02-18 at 4 38 05 PM" src="https://user-images.githubusercontent.com/34074929/74780182-0f4f7580-526d-11ea-99c0-5f8e22e773bd.png">
- When I click on Contact Us, I have to scroll past the left nav first before I see the Contact Us content 
<img width="511" alt="Screen Shot 2020-02-18 at 4 40 23 PM" src="https://user-images.githubusercontent.com/34074929/74780337-62292d00-526d-11ea-9719-5af0f8278a0b.png">

**Comment by cmajel** [comment]
These updates have been made:
General:
* Coloring updated
* Verified that the logo does go to home
* Cloud.gov casing on Get Started is fixed
* Sidenav now opens on FedRamp Authorized nav link
* Why cloud.gov > our team link updated

Mobile
* The hero image on the "Get Started" page is dropped 
* Sidenav moved to the bottom
* Pricing table has been updated

Screenshots: 
![Screen Shot 2020-02-21 at 3 39 54 PM](https://user-images.githubusercontent.com/2374206/75070085-8597e600-54c0-11ea-9e9b-96d8d9b43e4c.png)
![Screen Shot 2020-02-21 at 3 40 05 PM](https://user-images.githubusercontent.com/2374206/75070093-8a5c9a00-54c0-11ea-92d0-70eb47b055f6.png)
![Screen Shot 2020-02-21 at 3 40 15 PM](https://user-images.githubusercontent.com/2374206/75070101-8e88b780-54c0-11ea-8a64-9e9509b184c9.png)
